python/Sign_Extender.py

```diff
@@ -4,12 +4,12 @@
 def Sign_Extender(clk, instruction, output):
     #Instruction should be a 16-bit intbv
     #output is accurate 32-bit intbv.
-    output = intbv(max = 32)
+    output.Next = intbv(min = 32, max = 32)
```

Comment:
It looks like the initial code was mistaken, using intbv(mine = 32, max = 32) would constrain the output to the numbers 32 or 32. One would need to do "... = intbv(min = 0, max = 2**32) to get the correct range.